packages/open-payments/src/client/index.ts

```diff
@@ -129,6 +130,7 @@ const parseKey = (
 
 const createUnauthenticatedDeps = async ({
   useHttp = false,
+  validateResponses = true,
```

Comment:
the main change here: the addition of optional `validateResponses` flag when creating the open payments clients.